### PR TITLE
feat: LOS in explorer settings

### PR DESCRIPTION
**Pull request by pravusjif**
We enable LOS dynamic change from the explorer settings.

Testing link: 
https://play.decentraland.zone/branch/feat/LOSInExplorerSettings/index.html?ENV=org&position=-86%2C-135&realm=baldr-blue&renderer=urn%3Adecentraland%3Aoff-chain%3Arenderer-artifacts%3Afeat%2FLOSInExplorerSettings

Screen Shot 2021-05-13 at 12 12 51

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs [hunk 1]
```diff
@@ -110,6 +110,7 @@ private SettingsData.GeneralSettings GetDefaultGeneralSettings()
             {
                 sfxVolume = 1,
                 mouseSensitivity = 0.6f,
+                scenesLoadRadius = 10,
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs:113]: Default value should be 4, not 10.

Comment (pravusjif) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs:113]: in kernel master the LOS default is 10. 4 is the "secure radius":
https://github.com/decentraland/explorer/blob/444a95484ed484c5eaba47984aecbb19695d91bc/kernel/packages/decentraland-loader/lifecycle/manager.ts#L122

Comment (pravusjif) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs]: Corrected.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SliderSettingsControlView.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public override void Initialize(SettingsControlModel controlConfig, SettingsCont
             this.sliderControlConfig = (SliderControlModel)controlConfig;
             slider.maxValue = this.sliderControlConfig.sliderMaxValue;
             slider.minValue = this.sliderControlConfig.sliderMinValue;
-            slider.wholeNumbers = false;
+            slider.wholeNumbers = this.sliderControlConfig.wholeNumbers;
```
Comment (sandrade-dcl) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SliderSettingsControlView.cs:30]: Thanks for the fix! :)

**Review by sandrade-dcl — APPROVED** [review]
Reviewed and tested! Good job!

**Review by BrianAmadori — APPROVED** [review]

**Pull request merged**